fuel/utils/string_utils.py

```diff
@@ -0,0 +1,74 @@
+""" Utilities for modifying strings"""
+
+import os
+import re
+
+from theano.compat.six import string_types
+from theano.compat.six.moves import xrange
+
+from fuel.utils.exc import EnvironmentVariableError, NoDataPathError
+from fuel.utils.exc import reraise_as
+from fuel.utils.common_strings import environment_variable_essay
+
+
+def preprocess(string, environ=None):
```

Comment:
This was added for a reason. Before removing this we should see if that reason still apply.